redwood-treehouse-host/src/commonMain/kotlin/app/cash/redwood/treehouse/TreehouseApp.kt

```diff
@@ -46,9 +47,10 @@ public class TreehouseApp<A : AppService> private constructor(
 ) {
   private val codeHost = ZiplineCodeHost<A>()
 
-  public val dispatchers: TreehouseDispatchers = factory.dispatchers
+  // TODO: remove after https://github.com/cashapp/zipline/pull/1159
+  private val ziplineToPublisherHack = mutableMapOf<Zipline, RealEventPublisher>()
```

Comment:
I’m massively cheating here 'cause I want to correlate the Zipline-layer EventListener with the Treehouse-layer EventListener, but I don’t control the creation of the Zipline-layer one.

That PR has the fix!

Reply:
I’d like to hold this until I can rebase on that upgrade.